src/plugins/adjacent/adjacent.js

```diff
@@ -28,7 +28,19 @@ u.prototype.adjacent = function(html, data, callback) {
 
       return u(part).nodes;
     }).each(function(n){
-      fragment.appendChild(n);
+      var cloneWithEvents = n.cloneNode(true),
```

Comment:
I don't have much time now, but would it be possible to expand it with this?

```js
u(n).find('*').filter(function(node){ return node._e; }).each(function(){
  for ...
});
```

That way we also copy the children of `n` events, not only n's events. But it's getting really ugly really fast, so I totally agree with adding a `.clone()` method.